data/map systems.txt

```diff
@@ -8533,6 +8654,73 @@ system Cinxia
 			distance 270
 			period 16.6355
 
+system Citadelle
+	pos -980.667 225.917
+	government Pirate
+	arrival 1240
```

Comment:
The systems need system attributes. They should have at least ccor and south. You'll also want attributes like "bright star" and "notable star", using the same convention from #8474.
```suggestion
	government Pirate
	attributes "ccor" "south"
	arrival 1240
```